ui.apps/src/main/content/jcr_root/apps/testaem/components/customheader/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
+          xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+          jcr:primaryType="nt:unstructured"
+          jcr:title="Custom header"
+          sling:resourceType="cq/gui/components/authoring/dialog">
+    <content
+            jcr:primaryType="nt:unstructured"
+            sling:resourceType="granite/ui/components/coral/foundation/container"
+            maximized="{Boolean}true">
+        <items jcr:primaryType="nt:unstructured">
+            <tabs jcr:primaryType="nt:unstructured"
+                  sling:resourceType="granite/ui/components/coral/foundation/tabs">
+                <items jcr:primaryType="nt:unstructured">
+                    <internal-tab
+                            jcr:primaryType="nt:unstructured"
+                            sling:resourceType="granite/ui/components/coral/foundation/container"
+                            jcr:title="Text tab">
+                        <items jcr:primaryType="nt:unstructured">
+                            <text
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                    fieldLabel="Text"
+                                    name="./text"/>
+                        </items>
+                    </internal-tab>
+                    <internal-tab2
+                            jcr:primaryType="nt:unstructured"
+                            sling:resourceType="granite/ui/components/coral/foundation/container"
+                            jcr:title="Image tab">
+                        <items jcr:primaryType="nt:unstructured">
+                            <image
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/pathfield"
+                                    fieldLabel="Image"
+                                    name="./image"/>
+                        </items>
+                    </internal-tab2>
+                </items>
+            </tabs>
+            <!--            <columns jcr:primaryType="nt:unstructured"-->
```

Comment:
(cosmetic) don't forget to remove the commented code in the future. Commented code gets outdated quickly and is hard to maintain

Reply:
removed. 
corrected in the last commit.